### DISCUSSION TITLE
Issues with using the new Open WebUI interface, How to download models and enable GPU execution, How to set grok API?

### DISCUSSION THREAD

**Discussion by jasonsu123** [open-webui/open-webui#7308] [General]
Hello everyone,

1. I’ve downloaded the latest version of Ollama and installed Docker. However, the Open WebUI interface looks completely different from what was shown in older YouTube tutorials. Not only is it harder to use, but I also don’t know how to download models directly within Open WebUI.

In the settings, I can see a list of available models, but I can’t seem to paste commands to download them. It looks like it only allows importing JSON files, which feels even less user-friendly than using CMD.
![image](https://github.com/user-attachments/assets/bdd62a2c-bbc1-4b29-adbe-a14dff14c22f)

2. Additionally, could you explain the difference between these two commands?
 I’m currently using the first one—does this mean the models will only run on the CPU?
How can I configure Open WebUI to use the GPU?
When using Ollama in CMD, the speed is excellent, but Open WebUI feels very slow.

https://docs.openwebui.com/
If Ollama is on your computer, use this command:
docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main

To run Open WebUI with Nvidia GPU support, use this command:
docker run -d -p 3000:8080 --gpus all --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:cuda

Or, should I actually be using the command under "Installing Open WebUI with Bundled Ollama Support"?

3. Where can I configure the settings to use the currently free Grok API?

Thank you

**Comment by Andybeyond** [comment]
Yes experiencing the same problem with my new Open WebUI installation. 

Older version appears much easier to get a model downloaded through its interface, with intuitive UI. 

The new WebUI interface looks like a puzzle that's designed to confuse you. 

Downloading a model from the WebUI community used to be like a breeze. Now it makes you clueless. 

Feels like it's a piece of software no longer properly maintained.
